clients/gen/common.sh

```diff
@@ -17,7 +17,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-OPENAPI_GENERATOR_CLI_VER=5.4.0
+OPENAPI_GENERATOR_CLI_VER=7.0.1
```

Comment:
This will cause a lot of breaking changes to the RESTAPI. We should reserve it for the next major

Reply:
I am afraid @ephraimbuddy is right.

On the other hand this starts to be a little blocking. For instance for the go client, and other client issues where we need to manually fix things. (Missing supports for certain keywords in the spec etc...).

We didn't discuss this but maybe we could do a major release for client, independantly of airflow major release, just an idea, to not have clients stuck for a long long time.

Reply:
@pierrejeambrun i think mailing list discussion is required here

Reply:
Yeah. I think there are also another reason we should discuss it - namely security/release process for the clients. The "Python API" client we release might be seen as an "official ASF Source package"

Also there is one more point in general to discuss for that one, Currently we are voting on Python Client and we do not check the provenence of it - because this is code that got genereated by the release manager. and committed - unreviewable and unverifiable. 

So IMHO upgrading the client is also an opportunity to update the build and release process - what should happen, we (PMC members) should be able to verify that the build has been generated by the release manager from the API specification and not tampered with. It might be a "reproducible" build  or just "verifiable build" - but it boils down to being able to run the build and compare if what comes out of it (providing sufficient external tools we can rely-on - such as official image released by open-api). 

I think we can solve both problems at the same time - vy improving automation and adding simple-to-add verification step whether the package we are voting on is "really" generated from the specification without being tampered with.

This is part of the security improvement of our release process.